app/templates/main/record.html

```diff
@@ -59,49 +59,111 @@
+                                    <link nonce="{{ csp_nonce() }}"
+                                          href="https://cdn.jsdelivr.net/npm/universalviewer@4.0.25/dist/uv.min.css"
+                                          rel="stylesheet">
+                                    <noscript>
+                                        {% if file_extension == "pdf" %}
+                                            <object data="{{ url_for('static', filename='files/temp_file.pdf') }}"
+                                                    width="100%"
+                                                    height="700px">
+                                                <p>Your browser does not support PDFs. Please download the record to view it.</p>
+                                            </object>
+                                        {% elif file_extension == "png" or file_extension == "jpg" %}
+                                            <object data="{{ url_for('static', filename='files/temp_file.png') }}"
+                                                    width="100%"
+                                                    height="700px">
+                                                <p>Your browser does not support Images. Please download the record to view it.</p>
+                                            </object>
+                                        {% endif %}
+                                    </noscript>
+                                    <div id="uv" class="uv"></div>
+                                    <script id="init-uv" nonce="{{ csp_nonce() }}" manifest_url="{{ manifest_url }}" type="text/javascript" src="{{ url_for('static', filename='init.uv.js') }}">
+                                    </script>
+                                </div>
+                            </div>
                         {% endif %}
-                        <p class="govuk-body govuk-body--terms-of-use">
-                            Refer to <a href="/terms-of-use" class="govuk-link govuk-link--ayr">Terms of use.</a>
-                        </p>
-                    </div>
-                </div>
-                <div class="govuk-grid-column-one-third govuk-grid-column-one-third--record mobile-record-layout-record-arrangement">
-                    <div class="record-container">
-                        <h2 class="govuk-heading-m govuk-heading-m__record-header">Record arrangement</h2>
-                        <ol class="record-arrangement-list">
-                            {% for file_level in record["file_path"].split("/") %}
-                                <li class="govuk-body__record-arrangement-list">{{ file_level }}</li>
-                            {% endfor %}
-                        </ol>
+                        <!-- Record Details -->
+                        <div class="govuk-tabs__panel govuk-tabs__panel--hidden govuk-tabs__panel__record-details"
```

Comment:
could be worth extracting record details and record  view blocks into their own individual templates which we can include here